pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -369,6 +369,19 @@ func podMatchesAffinityTermProperties(pod *v1.Pod, properties []*affinityTermPro
 	return true
 }
 
+// podMatchesAnyAffinityTermProperties return true IFF the given pod matches any given property.
```

Comment:
Should be `returns true ...`

It reads like this time, it's not a "if and only if" case? i.e.`It returns true if the given pod matches any given ...`?

Reply:
Agree. `It returns true if the given pod matches any given ...` should be clear enough for this "any" case.

(leave previous `IFF` in `podMatchesAllAffinityTermProperties` as is. although I think `if` is also clear there.)